cognitedata/reveal
# feat(react-components): instance styling through asset mappings

#### Type of change

![Feat](https://img.shields.io/badge/Type-Feat-green)  

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/UX-1558

## Description :pencil:


Support instance styling using asset mappings

WIP: Currently, requests are sent every time you click, the plan is to cache this. 

## How has this been tested? :mag:



In HighlightNode storybook. Use model 
```
        modelId: 1791160622840317,
        revisionId: 502149125550840,
```

## Test instructions :information_source:



Click asset-mapped nodes (should be blue) to see that they are highlighted. Also check that there are no more requests sent when clicking again.


## Checklist :ballot_box_with_check:
